go/kbfs/libkbfs/interfaces.go

```diff
@@ -1903,6 +1903,9 @@ type InitMode interface {
 	// OldStorageRootCleaningEnabled indicates whether we should clean
 	// old temporary storage root directories.
 	OldStorageRootCleaningEnabled() bool
+	// DoRefreshCachedFavorites indicates whether we should refresh
+	// our cached versions of the favorites immediately upon a login.
+	DoRefreshCachedFavorites() bool
```

Comment:
Can we call this something that sounds more like initialization? e.g. `DoRefreshFavoritesOnInit`?